src/main/cpp/monero_jni_bridge.cpp

```diff
@@ -2239,6 +2344,1728 @@ JNIEXPORT jbyteArray JNICALL Java_monero_wallet_MoneroWalletFull_getCacheFileBuf
   }
 }
 
+
+//  ------------------------ FULL WALLET INSTANCE METHODS --------------------------
+
+JNIEXPORT jboolean JNICALL Java_monero_wallet_MoneroWalletLight_isViewOnlyJni(JNIEnv *env, jobject instance) {
```

Comment:
I would really like to avoid duplicating the JNI bridge code for every light wallet function.

Is it possible to not override the native functions in MoneroWalletLight, and for the existing JNI bridge functions to be called without duplication?